misc/N and its Double.c

```diff
@@ -0,0 +1,46 @@
+int main()
+{
+    int N,a,i,j,count=0;
+    printf("Enter N=");
+    scanf("%d",&N);
+    if(N>=2 & N<=500)
+    {
+        int A[500];
+        for(a=0;a<N;a++)
+        {
+            printf("Enter Value at A[%d]=",a);
+            scanf("%d",&A[a]);
+            if(A[a]<=103 & A[a]>=-103)
+            i++;
+            else
+            {
+                printf("Invalid Input");
+                exit(0);
+            }
+        }
+        for(i=0;i<N;i++)
+        {
+            for(j=0;j<N;j++)
+            {
+                if(i!=j)
+                {
+                    if(A[i]==2*A[j])
+                    {
+                        printf("True");
```

Comment:
You should prefer ending your ans with a new line character i.e. `\n`, make this change everywhere